x/stakeibc/keeper/icacallbacks_undelegate_test.go

```diff
@@ -287,86 +292,153 @@ func (s *KeeperTestSuite) TestGetLatestCompletionTime_Success() {
 	}
 	// Check that the second completion time (the later of the two) is returned
 	latestCompletionTime, err := s.App.StakeibcKeeper.GetLatestCompletionTime(s.Ctx(), txMsgData)
-	s.Require().NoError(err)
-	s.Require().Equal(secondCompletionTime.Unix(), latestCompletionTime.Unix())
+	s.Require().NoError(err, "get latest completion time succeeds")
+	s.Require().Equal(secondCompletionTime.Unix(), latestCompletionTime.Unix(), "latest completion time is the second completion time")
 }
 
 func (s *KeeperTestSuite) TestGetLatestCompletionTime_Failure() {
 	_ = s.SetupUndelegateCallback()
```

Comment:
nit: no need for assignment in these
```suggestion
	s.SetupUndelegateCallback()
```

Reply:
good catch, done